packages/runtime/client/rest/src/index.ts

```diff
@@ -244,14 +235,18 @@ export function mkRestClient(baseURL: string): RestClient {
     getContractById(contractId) {
       return unsafeTaskEither(Contract.getViaAxios(axiosInstance)(contractId));
     },
-    createContract(request) {
+    buildCreateContractTx(request) {
```

Comment:
Also @jhbertra, can you make this name change in the backend for a future release so we maintain the 1-1 parity?